docs/en/observability/synthetics-monitoring.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[synthetics-monitoring]]
+= Synthetics monitoring
+
+++++
+<titleabbrev>Synthetics monitoring</titleabbrev>
+++++
+
+[WARNING]
+====
+This functionality is experimental and may be changed or removed completely in a future release.
+Elastic will take a best effort approach to fix any issues, but experimental features are not subject
+to the support SLA of official GA features.
+
+Have a question? Want to leave feedback? Visit the
+https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics[Synthetics discussion forum].
+====
+
+[role="screenshot"]
+image::images/synthetic-app-overview.png[Synthetics app overview]
```

Comment:
This image was taken when the `url` links were broken. Could we swap it with a more recent capture that should have that fixed?

Reply:
Done.